docs/user-guide/sdks-using.md

```diff
@@ -65,6 +71,26 @@ Pull the packages from an online registry such as npm or PyPi.
 
     - To define versioning for Python packages, specify versions or version ranges in a `requirements.txt` file checked-in to your project. For more information, see [pip install](https://pip.pypa.io/en/stable/cli/pip_install/) in the pip documentation.
 
+### Install Java SDK from online registry
+
+To install this library in your project, use a build tool such as Maven, Gradle or Ant. Use the following link to get the necessary artifact:
+
+https://mvnrepository.com/artifact/org.zowe.client.java.sdk/zowe-client-java-sdk
+
+For a Maven project add the SDK as a dependency by updating your pom.xml as follows:
+
+<dependency>
```

Comment:
These should be turned into code blocks:

```xml
<dependency>
    <groupId>org.zowe.client.java.sdk</groupId>
    <artifactId>zowe-client-java-sdk</artifactId>
    <version>2.2.0</version>
</dependency>
```

```groovy
implementation group: 'org.zowe.client.java.sdk', name: 'zowe-client-java-sdk', version: '2.2.0'
```

Reply:
I tabbed the content and it now shows in a highlighted block. I hope that is good enough. See latest checkin.